services/mgmt/activedirectory/Cargo.toml

```diff
@@ -46,5 +46,4 @@ default_tag = ["package-2020-03"]
 "package-2017-04-01" = []
 
 [lints]
-workspace = true
-
+workspace = true
```

Comment:
Missing cr at end of file?

Reply:
Odd. The template has a separate line break but the process of writing the file must've removed it. It's possible this file was modified manually at some point to fix a build break. I'm not going to fret about it, though.